inclusion/blob_share_commitment_rules_test.go

```diff
@@ -257,25 +258,35 @@ func TestRoundUpByMultipleOf(t *testing.T) {
 	}
 }
 
+type roundUpTestCase struct {
+	input int
+	want  int
+}
+
+var roundUpTestCases = []roundUpTestCase{
+	{input: -1, want: 1},
+	{input: 0, want: 1},
+	{input: 1, want: 1},
+	{input: 2, want: 2},
+	{input: 4, want: 4},
+	{input: 5, want: 8},
+	{input: 8, want: 8},
+	{input: 11, want: 16},
+	{input: 511, want: 512},
+	{input: math.MaxInt32 - 1, want: 1 << 31},
+	{input: math.MaxInt32 + 1, want: 1 << 31},
+	{input: math.MaxInt32, want: 1 << 31},
+	{input: math.MaxInt >> 1, want: 1 << 62},
+	{input: math.MaxInt, want: 1},
+}
```

Comment:
[blocking] these were previously defined inside the function `TestRoundUpPowerOfTwo`. Can you please move them back there so that their scope is confined to that function